.gitignore

```diff
@@ -1,8 +1,5 @@
 /palma.db
 /palma.ini
 /uploads
-/palma.po
```

Comment:
This was a debug change I made, when I could not get my modified language file to be pulled and applied on my test machine. I probably forgot to undo it. I did so now.